src/cryptography/x509/extensions.py

```diff
@@ -342,6 +342,9 @@ def __eq__(self, other):
     def __ne__(self, other):
         return not self == other
 
+    def __hash__(self):
+        return hash(self.ca) ^ hash(self.path_length)
```

Comment:
:+1:

On Sun, Dec 27, 2015 at 2:00 PM, Paul Kehrer <notifications@github.com>
wrote:

> In src/cryptography/x509/extensions.py
> <https://github.com/pyca/cryptography/pull/2594#discussion_r48457325>:
>
> > @@ -342,6 +342,9 @@ def __eq__(self, other):
> >      def __ne__(self, other):
> >          return not self == other
> >
> > +    def __hash__(self):
> > +        return hash(self.ca) ^ hash(self.path_length)
>
> Seems like hashing the tuple and letting the underlying implementation xor
> is reasonable. I'll switch it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pyca/cryptography/pull/2594/files#r48457325>.
>



-- 
"I disapprove of what you say, but I will defend to the death your right to
say it." -- Evelyn Beatrice Hall (summarizing Voltaire)
"The people's good is the highest law." -- Cicero
GPG Key fingerprint: 125F 5C67 DFE9 4084